src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -53,30 +57,27 @@
 import ch.njol.skript.config.Node;
 import ch.njol.skript.config.SectionNode;
 import ch.njol.skript.lang.Variable;
+import ch.njol.skript.log.SkriptLogger;
 import ch.njol.skript.registrations.Classes;
 import ch.njol.skript.registrations.Converters;
-import ch.njol.skript.variables.DatabaseStorage.Type;
 import ch.njol.skript.variables.SerializedVariable.Value;
 import ch.njol.util.Closeable;
 import ch.njol.util.Kleenean;
 import ch.njol.util.NonNullPair;
 import ch.njol.util.SynchronizedReference;
 import ch.njol.yggdrasil.Yggdrasil;
 
-/**
- * @author Peter Güttinger
- */
 public abstract class Variables {
-	private Variables() {}
-	
+
 	public final static short YGGDRASIL_VERSION = 1;
-	
 	public final static Yggdrasil yggdrasil = new Yggdrasil(YGGDRASIL_VERSION);
 
-	public static boolean caseInsensitiveVariables = true;
-	
-	private final static String configurationSerializablePrefix = "ConfigurationSerializable_";
+	private final static Multimap<String, Class<? extends VariablesStorage>> types = HashMultimap.create();
```

Comment:
How come this is a Multimap?

Reply:
There can be multiple databases because of the patterns.